com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKPokeReticleVisual.cs

```diff
@@ -38,20 +38,23 @@ protected void OnDisable()
         [BeforeRenderOrder(XRInteractionUpdateOrder.k_BeforeRenderLineVisual)]
         private void UpdateReticle()
         {
-            using (UpdateReticlePerfMarker.Auto())
+            if(Reticle != null)
             {
-                Reticle.SetActive(pokeInteractor.enabled && pokeInteractor.isHoverActive);
-
-                // TODO: Ideally we'd want the ReticleVisualScript to be responsible for all aspects of a visuals appearance.
-                // This is because it would allow us to freely use different reticle icons without changing other behavior.
-                // However, until the ReticleMagnetism class is revisited and potentially incorporated into this class, we'll
-                // leave all pose related responsibilities to the ReticleMagnetism component.
-                // reticle.transform.SetPositionAndRotation(pokeInteractor.PokeTrajectory.End, pokeInteractor.attachTransform.rotation);
-
-                // The proximity light should only be active when the reticle is
-                if (proximityLight != null)
+                using (UpdateReticlePerfMarker.Auto())
```

Comment:
nit: we might want to include the null check inside the perf marker, just so _any_ work this method is doing is accounted for when it's called. It might look weird otherwise in the profiler? But I'm not sure

Reply:
Fixed